web/components/App.jsx

```diff
@@ -106,12 +106,8 @@ class App extends React.Component {
             });
         });
       } else {
-        this.assignParticipants(roomId, role.label, values).then(() => {
-          bdk.createBotData(this.props.roomId, botName, 'participants' + role.label,
```

Comment:
I am assuming assignParticipants (109) created the new bot data and line 110 is creating a separated bot data for the role, right?

Reply:
Usually, this line of code creates the botData if it doesn't exist (i.e. when the bot loaded the first time in that room). 
Need to double check if we can remove the 'createBotData'. Will new rooms work (botData created as expected)?

Reply:
yeah, line 110 was creating a separate BotData for each Assigned participant Role botData per each assingned participant -> Role (the old way)... with the new `assignedParticipants` botData. (which is now created at init()). We're now removing this createBotData as the plan is to deprecated it. the `.then` make sure we still dispatch the event so that the other bots can receive the update with the new assigned Participants.

Reply:
@LongweiDeng they should work, once we release the [RoomType PR](https://github.com/SFDC/refocus-roomtypes/pull/85), which will only use the new Bot data `assignedParticipants`.